scripts/filtered-properties-next-to-each-pom.groovy

```diff
@@ -0,0 +1,36 @@
+import java.util.stream.Collectors
+
+/**
+ * Contents of test.properties file to be written. Contains key=value entries.
+ * Only subset of system properties is written, based on the provided comma separated list
+ * of properties.
+ * @param inputList subset of system properties to be written to file
+ * @return String with contents of the file
+ */
+private String filteredPropertiesFileContents(String inputList) {
```

Comment:
Suggestion : how about reducePropertiesFileContents()?

Reply:
I simplified to `getTestPropertiesFileContents` name of the method while the script now is named `write-test-properties-file-for-invoker.groovy`. Filtering is now an optional operation, if no value for `listedProperties` argument is passed to script then  all userProperties are written.